app/[locale]/layout.tsx

```diff
@@ -38,6 +39,7 @@ export default function RootLayout({
         <Header locale={locale} />
         <section className="grow">{children}</section>
         <Footer locale={locale} />
+        <Toaster /> {/*temp for preview*/}
```

Comment:
Will this be removed? Seems like this sets where the toast is displayed?

Reply:
It's about we want it scoped, it should be here by default yeah

Reply:
So we can remove it?

Reply:
It acts as an anchor to where the toast will render, the only downside of leaving it here is that it will load once (js not html) for everyone who visits the website.

Reply:
But then what did you mean "it should be here by default"

Reply:
If it's in the root layout any page on the website can use toast freely hence it being the default layout,
if we want it to be accessed by only certain pages we can move it up the ladder to target their specific layout.